lib/liquid/context.rb

```diff
@@ -74,32 +74,13 @@ def pop_interrupt
     end
 
     def handle_error(e, line_number = nil, raw_token = nil)
-      if e.is_a?(Liquid::Error)
-        e.template_name ||= template_name
-        e.line_number ||= line_number
-      end
-
-      output = nil
-
-      if exception_handler
-        args = [e]
-        args << { line_number: line_number, raw_token: raw_token } if exception_handler.arity == 2
```

Comment:
This removes the token from the error which was added for debugging, can we add that back?